addons/mass_mailing/tests/common.py

```diff
@@ -187,6 +187,7 @@ def assertMailTraces(self, recipients_info, mailing, records,
                             content=content,
                             email_to_recipients=email_to_recipients,
                             fields_values=fields_values,
+                            email_values=email_values,
```

Comment:
I know those tools are a bit *bordéliques* (anyway I am the one who use them generally so voila :D ), but ``email_values`` should be given as a value of ``recipient_info`` (which is a item of ``recipients_info`` list). Idea is that you give a list of data to check, you could check for failed and sent emails for example, with different email_values. It has to be added in the ``invalid`` computation at iteration beginning.

Reply:
This code is used in many places and I tried to change as little as possible. `assertMailMailWId` and `assertMailMailWEmails` already accepted that argument instead of a recipient info or an author. I want to avoid a bigger refactor here.